libs/core-ui/src/lib/Interfaces/DataBalanceInterfaces.ts

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export interface IDataBalanceMeasures {
+  aggregateBalanceMeasures?: IAggregateBalanceMeasures;
+  distributionBalanceMeasures?: IDistributionBalanceMeasures;
+  featureBalanceMeasures?: IFeatureBalanceMeasures;
+}
+
+export interface IAggregateBalanceMeasures {
+  measures: { [measureName: string]: number };
+}
+
+export function getAggregateBalanceMeasures(measures: {
+  [measureName: string]: number;
+}): { [measureName: string]: number } {
+  return measures;
+}
+
+export interface IDistributionBalanceMeasures {
+  features: string[];
+  measures: { [featureName: string]: { [measureName: string]: number } };
+}
+
+interface DistributionMeasureInfo {
+  name: string;
+  description?: string;
+}
+
+export const ApprovedDistributionMeasures = new Map<
+  string,
+  DistributionMeasureInfo
+>([
+  ["inf_norm_dist", { name: "Infinite Norm Distance" }],
+  ["js_dist", { name: "Jensen–Shannon Distance" }],
+  ["kl_divergence", { name: "Kullback-Leibler Divergence" }],
+  ["total_variation_dist", { name: "Total Variation Distance" }],
+  ["wasserstein_dist", { name: "Wasserstein Distance" }]
+]);
+
+export function getDistributionBalanceMeasures(
+  measures: { [featureName: string]: { [measureName: string]: number } },
+  featureName: string
+): { [measureName: string]: number } {
+  if (featureName in measures) {
+    return measures[featureName];
+  }
+
+  return {};
+}
+
+export interface IFeatureBalanceMeasures {
+  uniqueClasses: { [featureName: string]: string[] };
+  features: string[];
+  measures: {
+    [featureName: string]: {
+      [classKey: string]: { [measureName: string]: number };
+    };
+  };
+}
+
+interface FeatureBalanceMeasureInfo {
+  varName: string;
+  description: string;
+  range?: [number, number];
+}
+
+// The key is the formatted name instead of var name because its keys are used to populate the Measure Dropdown
+export const ApprovedFeatureBalanceMeasures = new Map<
+  string,
+  FeatureBalanceMeasureInfo
+>([
+  [
+    "Demographic Parity",
+    {
+      description:
```

Comment:
all of these descriptions need to go into the localization files

Reply:
since you do this in line highlighting:
<b>Demographic Parity</b>
I wonder if you should break apart the terms in the localization.  Not sure how this will work with different languages.  Perhaps it might be better to remove the highlights in text.